### ISSUE TITLE
Mobile interaction

### ISSUE THREAD

**Issue by LJWatson** [bkardell/common-panel#28]
Only basic events are available on touch devices when an AT is enabled. When an AT is enabled most gestures fire mouse events (not keyboard events), except in some circumstances when blur/focus events are fired instead.

For details:
http://patrickhlauke.github.io/touch/tests/results/#mobile-tablet-touchscreen-assistive-technology-events 

This may only effect the prototype examples (we should test them on mobile if that hasn't happened already). It may be something we need to think about in terms of native implementation as well though?

**Comment by chaals** [comment]
Assuming we get this into HTML, we need to understand how browsers can make it possible to interact, but I don't think we want to tell them how they have to do it.

So we need feedback if it can't be done, but otherwise it should be fine.

And for that matter, we should be defining the "responsive events" that need to work - how to interact with the thing in the first place without linking it to any particular interaction behaviour except to show there is a feasible implementation strategy.

**Comment by LJWatson** [comment]
@Chaals

Agreed. FWIW I think this is only something we need to consider for the prototypes. The mobile interaction thing is only an issue when scripting is used to handle the interaction - because it tends to rely on keyboard events which aren't always available on mobile + AT.
